h/api/search/query.py

```diff
@@ -96,8 +96,18 @@ class AuthFilter(object):
     user's effective principals will pass through this filter.
     """
 
-    def __init__(self, request):
+    def __init__(self, request, private=True):
+        """Initialize a new AuthFilter.
+
+        :param request: the pyramid.request object
+
+        :param private: whether or not to include private annotations in the
+            search results
+        :type private: bool
+
+        """
         self.request = request
+        self.private = private
 
     def __call__(self, params):
         groups = list(self.request.effective_principals)
```

Comment:
This initially confused me because 'groups' is not in fact, just a list of groups but a set of `principals` which can include both groups and user IDs. I suggest renaming the `groups` variable here accordingly (eg. to `principals`)